### ISSUE TITLE
Linkdrop Contract Standard 

### ISSUE THREAD

**Issue by Dobrokhvalov** [LinkdropProtocol/linkdrop-monorepo#3]
Contract Interface: 
- [x] claimToken
- [x] claimNFT
- [x] checkParams

**Comment by Dobrokhvalov** [comment]
Outdated one-to-one interface for example: 

```
contract ILinkdrop {
    event Deposited(address _key, address sender, address tokenAddress, uint _amount, uint _expirationTime)
    event Cancelled(address _key, address sender, address tokenAddress, uint _amount, uint _expirationTime)
    event Claimed(address _key, address sender, address tokenAddress, uint _amount, uint _expirationTime)
    
    // checks 
    function isSignatureValid(address _key, address _receiver, bytes memory _signature) public view returns (bool result) {}
    function isClaimed(address _key, address _receiver, bytes memory _signature) public view returns (bool result) {}
    function isClaimValid(address _key, address _receiver, bytes memory _signature) public view returns (bool result) {}
    function isLinkExpired(address _key) {}
    
    
    function getLinkdrop(address _key) public view returns(address sender, address tokenAddress, uint _amount, uint _expirationTime) {}
    
    
    // if not expired only sender can cancel the linkdrop 
    // otherwise any call call 
    function cancel(address _transitAddress) public returns (bool success) {} 
    
    function claim(address _key, address _receiver, bytes memory _signature) public {}   
}
```

**Comment by amiromayer** [comment]
Submitted PR with interfaces directory in contracts package

**Comment by Dobrokhvalov** [comment]
Implemented in #5
